openid-4-verifiable-presentations-1_0.md

```diff
@@ -616,11 +624,325 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
+of claims and Credentials, it does not define details about the user interface
+of the Wallet, for example, if and how users can select which combination of
+Credentials to present. However, it is typically expected that the Wallet
+presents the End-User with a choice of which Credential(s) to present if
+multiple of the sets of Credentials in `options` can satisfy the request.
+
+#### Security Considerations {#dcql_query_security}
+
+While the Verifier can specify various constraints both on the claims level and
+the Credential level as shown above, it MUST NOT rely on the Wallet to enforce
+these constraints. The Wallet is not controlled by the Verifier and the Verifier
+MUST perform its own security checks on the returned Credentials and
+presentations.
+
+## Format-specific Properties {#format_specific_properties}
+
+OpenID for Verifiable Presentations is Credential Format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any Credential Format. This section defines Credential Format Profiles for some of the known Credential Formats. Other specifications or deployments can define their own Credential Format Profiles:
+
+### IETF SD-JWT VC {#format_vc_sd_jwt}
+
+`vct_values`:
+: OPTIONAL. An array of strings that specifies allowed values for
+the type of the requested Verifiable Credential. All elements in the array MUST
+be valid type identifiers as defined in [@!I-D.ietf-oauth-sd-jwt-vc]. The Wallet
+MAY return credentials that inherit from any of the specified types, following
+the inheritance logic defined in [@!I-D.ietf-oauth-sd-jwt-vc].
+
+### Mobile Documents or mdocs (ISO/IEC 18013 and ISO/IEC 23220 series) {#format_mso_mdoc}
+
+`doctype_value`:
+: OPTIONAL. String that specifies an allowed value for the
+doctype of the requested Verifiable Credential. It MUST
+be a valid doctype identifier as defined in [@ISO.18013-5].
+
+
+## Claims Path Pointer {#claims_path_pointer}
+
+A claims path pointer is a pointer into the JSON structure of the Verifiable
+Credential, identifying one or more claims. A claims path pointer MUST be a
+non-empty array of strings and non-negative integers. A string value 
+indicates that the respective key is to be selected, the special string value `*` 
+indicates that all elements of the currently selected array(s) are to be selected;
+and a non-negative integer indicates that the respective index in an array is to be selected. The path
+is formed as follows:
+
+  - Start with an empty array.
+  - To address a particular claim within an object, append the key (claim name)
+    to the array.
+  - To address an element within an array, append the index to the array (as a
+    non-negative, 0-based integer).
+  - To address all elements within an array, append the string `*` to the array.
+
+Verifiers MUST NOT point to the same claim more than once in a single query.
+Wallets SHOULD ignore such duplicate claim queries.
+
+### Example
```

Comment:
Dear,
I will humbly submit that this example would go better after `### Processing` so that the processing rules about selection which the example exemplifies are seen before said example.
With Most Entitled Respect,

Reply:
Dear,
Done.
-Dr. Daniel